src/MailViewer.php

```diff
@@ -83,8 +102,7 @@ public static function prepareMails(array $mailables): array
 
             if ($constructorParameters !== $givenParameters) {
                 throw new Exception(
-                    "The arguments passed for {$mailable} in the config/mailviewer.php file do not match with the constructor 
-                    params of the {$mailable} class or the constructor params of the {$mailable} class aren't typehinted"
+                    "The arguments passed for {$mailable} in the config/mailviewer.php file do not match with the constructor params of the {$mailable} class or the constructor params of the {$mailable} class aren't typehinted"
```

Comment:
Can you revert this change so that we adhere to PSR2's guideline for length of the line being 120 chars max. Also, we won't need to horizontally scroll when interacting with this file on GitHub if done 😄